src/redux/books/books.js

```diff
@@ -0,0 +1,17 @@
+export const addBooks = (newBook) => ({ type: 'ADD', newBook });
+export const removeBooks = (id) => ({ type: 'REMOVE', id });
+
+const bookReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'ADD':
+      return [...state, action.newBook];
+
+    case 'REMOVE':
+      return [...state].filter((book) => book.id !== action.id);
+
+    default:
+      return state;
+  }
+};
+
```

Comment:
- [OPTIONAL] Kindly make the action type a const variable and use it in action creator and reducer instead of string to not repeat yourself and if you decide to change the action name you need only to change it in one place.